hail/python/hail/docs/change_log.md

```diff
@@ -1,5 +1,26 @@
 # Change Log
 
+## 0.2.18
+
+Released 2019-07-12
+    
+### Critical performance bug fix
+
+- (hail#6605) Resolved code generation issue leading a performance 
+  regression of 1-3 orders of magnitude in Hail pipelines using
+  constant strings or literals. This includes almost every pipeline!
+  **This issue has exists in versions 0.2.15, 0.2.16, and 0.2.17, and
+  any users on those versions should update as soon as possible.
```

Comment:
This needs a `**` at the end, doesn't it?

Reply:
yep